### PR TITLE
JAMES-1950 Adding a metric for mail queue size

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: server/queue/queue-jms/src/main/java/org/apache/james/queue/jms/JMSMailQueue.java [hunk 1]
```diff
@@ -246,6 +250,7 @@ protected void produceMail(Session session, Map<String, Object> props, int msgPr
                 message.setObjectProperty(entry.getKey(), entry.getValue());
             }
 
+            mailQueueSize.decrement();
```
Comment (rouazana) [inline server/queue/queue-jms/src/main/java/org/apache/james/queue/jms/JMSMailQueue.java]: I don't understand why you decrement here.

Comment (chibenwa) [inline server/queue/queue-jms/src/main/java/org/apache/james/queue/jms/JMSMailQueue.java]: It's a fail. Corrected

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**